pages/treeherder.py

```diff
@@ -51,6 +64,12 @@ def open_perfherder_page(self):
         from perfherder import PerfherderPage
         return PerfherderPage(self.base_url, self.selenium).wait_for_page_to_load()
 
+    def open_single_resultset(self):
+        Wait(self.selenium, self.timeout).until(
+            EC.visibility_of_element_located(self._first_resultset_datestamp_locator))
+        self.selenium.find_element(*self._first_resultset_datestamp_locator).click()
+        return self.selenium.find_element(*self._single_resultset_datestamp_locator)
```

Comment:
You don't need to return anything here. The only thing that might be worth returning is a page object for the single result page, for which you could just return `self` as it's the same as the existing page (but with less results).